flink-runtime/src/test/java/org/apache/flink/runtime/dispatcher/DispatcherHATest.java

```diff
@@ -150,8 +154,22 @@ public void testGrantingRevokingLeadership() throws Exception {
 	}
 
 	@Nonnull
-	private HATestingDispatcher createHADispatcher(TestingHighAvailabilityServices highAvailabilityServices, BlockingQueue<DispatcherId> fencingTokens) throws Exception {
+	private HATestingDispatcher createDispatcher(TestingHighAvailabilityServices highAvailabilityServices, Queue<DispatcherId> fencingTokens) throws Exception {
```

Comment:
I would suggest to group all helper methods after tests.

Reply:
Good point. Will do it.